### ISSUE TITLE
[demonoid] Found no results while trying to browse this tracker (Test)

### ISSUE THREAD

**Issue by squirrelboot** [Jackett/Jackett#3136]
**full enhanced log IS included**

**Jackett version**: 0.8.1000.0
**Mono version** (if not using Windows):  Mono JIT compiler version 5.12.0.226

-------------------------------
Hello,

My login information is accurate, but for some reason it is not working with Demonoid. 
Error states:  

> An error occured while testing this indexer
> Found no results while trying to browse this tracker

Enhanced logs were enabled and are attached. 

[log.txt](https://github.com/Jackett/Jackett/files/2030976/log.txt)

**Comment by kaso17** [comment]
The log only contains demonoid searchs without enhanced logging. jackett was restarted multiple times in your log (enhanced logging enabled via the GUI is reset on a restart).

**Comment by squirrelboot** [comment]
Ah, I did not realize it is reset on a restart. I enabled it and did a few things to try and push that info into logs. Is this better information?

[log.txt](https://github.com/Jackett/Jackett/files/2031332/log.txt)

**Comment by kaso17** [comment]
```
2018-05-23 16:51:59.3569 Debug WebClient(HttpWebClient).GetString(Url:https://www.demonoid.pw/files/?category=0&subcategory=All&quality=All&seeded=2&to=1&query=&external=2) 
2018-05-23 16:51:59.8477 Debug [MONO relative redirect bug] Rewriting relative redirect URL from file:///account_handler.php%3Freturnpath=%252Ffiles%252F%253Fcategory%253D0%2526subcategory%253DAll%2526quality%253DAll%2526seeded%253D2%2526to%253D1%2526query%253D%2526external%253D2&withq=1 to https://www.demonoid.pw/account_handler.php%3Freturnpath=%252Ffiles%252F%253Fcategory%253D0%2526subcategory%253DAll%2526quality%253DAll%2526seeded%253D2%2526to%253D1%2526query%253D%2526external%253D2&withq=1 
2018-05-23 16:51:59.8502 Debug WebClient(HttpWebClient): Returning MovedPermanently => https://www.demonoid.pw/account_handler.php%3Freturnpath=%252Ffiles%252F%253Fcategory%253D0%2526subcategory%253DAll%2526quality%253DAll%2526seeded%253D2%2526to%253D1%2526query%253D%2526external%253D2&withq=1  
```
For some reason it's trying to redirect you to account_handler.php. Please try to re-configure it.

**Comment by squirrelboot** [comment]
Ok, so I had not checked it since posting the log. I just tried it now and it suceeded without me changing anything. What I am thinking is that maybe after so many attempts at connecting with the wrong information.. it had a cool down period of some kind before it would work again. I had fixed the login issue yesterday, but it still didn't work. So this is the only thing I can think of. A cool down period. 

It is currently working. Thank you very much for taking the time to look through my log and try to assist me. It's honestly very appreciated. I wasn't sure what was going on and it was driving me up a wall.
